library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -871,12 +871,16 @@ public void notifyStateChangedToLoaded(Section section, State previousState) {
             }
         }
 
-        notifyItemChangedInSection(section, 0);
-
         int contentItemsTotal = section.getContentItemsTotal();
 
-        if (contentItemsTotal > 1) {
-            notifyItemRangeInsertedInSection(section, 1, contentItemsTotal - 1);
+        if (contentItemsTotal == 0) {
+            notifyItemRemovedFromSection(section, 0);
```

Comment:
I'd rather not take any action in this case, developer should call the correct notify method